pkg/translator/testdata/otlp_with_transform_collector_config.yaml

```diff
@@ -0,0 +1,41 @@
+receivers:
+    otlp/otlp_in:
+        protocols:
+            grpc:
+                endpoint: ${STRAWS_COLLECTOR_POD_IP}:9922
+            http:
+                endpoint: ${STRAWS_COLLECTOR_POD_IP}:1234
+processors:
+    batch: {}
+    transform/my_transformer:
+        error_mode: ignore
+        logs_statements:
+            - some_logs_transform_statement
+            - another_logs_transform_statement
+        metrics_statements:
+            - some_metrics_transform_statement
+            - another_metrics_transform_statement
+        traces_statements:
+            - some_traces_transform_statement
+            - another_traces_transform_statement
```

Comment:
Do we need to support both [basic](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor#basic-config) and [advanced](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor#advanced-config) configuration?